pkg/deploy/config.go

```diff
@@ -67,6 +67,7 @@ type Configuration struct {
 	GatewayFeatures                    []string      `json:"gatewayFeatures,omitempty"`
 	GatewayMDSDConfigVersion           *string       `json:"gatewayMdsdConfigVersion,omitempty" value:"required"`
 	GatewayStorageAccountDomain        *string       `json:"gatewayStorageAccountDomain,omitempty" value:"required"`
+	GatewayVMSize                      *string       `json:"gatewayVmSize,omitempty" value:"required"`
```

Comment:
Because we have a default value, this doesn't need `value:"required"` now as it should default to `Standard_D8s_v3`.

Reply:
Fixed.